desktop/apps/consignments2/client/actions.js

```diff
@@ -14,12 +35,42 @@ export function createSubmission () {
   }
 }
 
+export function fetchArtistSuggestions (value) {
+  return async (dispatch, getState) => {
+    try {
+      request.get(`${sd.API_URL}/api/v1/match/artists`)
```

Comment:
Super agent is still a great request library and I don't think we need to depart with it—that said the new native `fetch` API (and it's [pollyfill](https://github.com/matthew-andrews/isomorphic-fetch)) is becomming more popular these days.

As for mocking this in tests, a combination of rewire and sinon should do the trick...

````javascript
let request

beforeEach(() => {
  request = sinon.stub()
  request.get = sinon.stub().returns(request)
  request.query = sinon.stub().returns(request)
  request.set = sinon.stub().returns(request)
  request.end = sinon.stub().returns(request)
  module.__set__('request', request)
})
````